taipy/gui/utils/_bindings.py

```diff
@@ -39,6 +39,8 @@ def _bind(self, name: str, value: t.Any) -> None:
 
     def __get_property(self, name):
         def __setter(ud: _Bindings, value: t.Any):
+            if isinstance(value, _MapDict):
```

Comment:
😗

Reply:
Oh whao

Reply:
I have tested it and there should not be any obvious one since it is a setter and the update_var method is called right below it